superset/assets/src/explore/controls.jsx

```diff
@@ -951,6 +951,7 @@ export const controls = {
     freeForm: true,
     label: t('Time range'),
     default: t('Last week'),
+    description: t('The time range for the visualization. Client times are expressed using UTC as the local time.'),
```

Comment:
wording nit: I prefer `The time range for the visualization. Times are in UTC`. Gets the same point across but is much more succinct